### PR TITLE
create a pipeline with provision env task

### PR DESCRIPTION
**Pull request by kasemAlem**
As part of story [KONFLUX-2457](https://issues.redhat.com/browse/KONFLUX-2457) , this PR 
is about creating a new integration test scenario pipeline yaml file to include the new task of creating environment to run the ITS PLR .

### CODE REVIEW

File: pipelines/integration_resolver_pipeline_environment_pass.yaml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+      params:
+        - name: RESULT
+          value: SUCCESS
+        - name: KONFLUXNAMESPACE
+          value: $(context.pipelineRun.namespace)
+        - name: PIPELINERUN_NAME
+          value: $(context.pipelineRun.name)
+        - name: PIPELINERUN_UID
+          value: $(context.pipelineRun.uid)
+    - name: task-success-2
```
Comment (jsztuka) [inline pipelines/integration_resolver_pipeline_environment_pass.yaml:39]: What is the idea behind this second success task? I see its the same as task-success one.

Comment (kasemAlem) [reply pipelines/integration_resolver_pipeline_environment_pass.yaml:39]: no specific idea, just cloned `pipelines/integration_resolver_pipeline_pass.yaml` then I've added the new task  that take care of environment flow

Comment (dirgim) [reply pipelines/integration_resolver_pipeline_environment_pass.yaml:39]: My vote is to skip this task to keep the pipeline focused on the important tasks.

File: pipelines/integration_resolver_pipeline_environment_pass.yaml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+          - name: url
+            value: https://github.com/konflux-ci/integration-examples 
+          - name: revision
+            value: main
+          - name: pathInRepo
+            value: tasks/test_output.yaml
+      params:
+        - name: RESULT
+          value: SUCCESS
+    - name: task-skipped
```
Comment (dirgim) [inline pipelines/integration_resolver_pipeline_environment_pass.yaml]: My vote is to skip this task also to keep the pipeline focused on the important tasks for environment provisioning.

**Review by jsztuka — APPROVED** [review]

File: pipelines/integration_resolver_pipeline_environment_pass.yaml (hunk 2 above)
Comment (dirgim) [reply pipelines/integration_resolver_pipeline_environment_pass.yaml]: I'm still seeing this task, I meant that it should be removed from this pipeline.

File: pipelines/integration_resolver_pipeline_environment_pass.yaml [hunk 3]
```diff
@@ -0,0 +1,64 @@
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: integration-resolver-pipeline-pass
```
Comment (dirgim) [inline pipelines/integration_resolver_pipeline_environment_pass.yaml]: Let's name this something in line with it's use-case, like `integration-environment-pipeline` or similar.

Comment (kasemAlem) [reply pipelines/integration_resolver_pipeline_environment_pass.yaml]: that sounds much better,  I picked the below if no objection 

`  name: integration-environment-pipeline-pass`

**Review by dirgim — APPROVED** [review]
LGTM

**Pull request merged**